src/sql/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -310,29 +310,27 @@ export class NotebookEditor extends BaseEditor implements IFindNotebookControlle
 					);
 					this._setCurrentFindMatch(findScope);
 				} else {
-					this.notebookInput.notebookFindModel.clearDecorations();
```

Comment:
I'm a little concerned around the changes in the product code here. Can you give some info as to the motivation behind these changes

Reply:
This change has no functional implication. It is just changing code body from within a 'then' block to code coming after await. The form that I have changed to is just more readable and is our recommended form for async calls. if there is any concern, I am ok with doing any additional testing to address any risks.

Reply:
Oh, I see that the call to clearDecorations() is gone. was that intentional?

Reply:
no that was not. I will double check on that.

Reply:
Yes indeed! I seem to have inadvertently deleted the call to clearDecorations(). Thanks for catching that!.

Reply:
added back the call to clearDecorations()